source/push-to-deploy.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Push To Deploy
+order: 19
+description: Learn how to setup Push to Deploy on Cloud, so you can deploy changes to your apps just buy by performing a commit to a repository.
+---
+
+With this feature, never was easier to create an app on Galaxy. All you need to do is follow three  simple steps, then you'll have a new app deployed.
+
+<h2 id="pre-requirements">Pre requirements</h2>
+
+Today we have our integration just with GitHub. In the future we'll add more Git providers.
+
+The only requirement you need today to start using this feature is to have a repository on GitHub with a Meteor project on it. And that's it.
```

Comment:
We don't support all versions of Meteor, we should explain this here.

Reply:
:+1: